docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -411,8 +366,8 @@ RUN powershell add-windowsfeature web-asp-net45
 
 ### Additional resources
 
--   **aspnet-docker/Dockerfile.** Example Powershell commands to run from dockerfiles to include Windows features.
-    [*https://github.com/Microsoft/aspnet-docker/blob/master/4.6.2/Dockerfile*](https://github.com/Microsoft/aspnet-docker/blob/master/4.6.2/Dockerfile)
+- Example Powershell commands to run from dockerfiles to include Windows features -
+    [aspnet-docker/Dockerfile](https://github.com/Microsoft/aspnet-docker/blob/master/4.6.2/Dockerfile)
```

Comment:
there are some things that are special about the e-books @gewarren. like in this case because the content is also printed in a book, some urls are displayed in full in the additional resources list